### ISSUE TITLE
Add nightmarejs in JavaScript section

### ISSUE THREAD

**Issue by xcarpentier** [BruceDone/awesome-crawler#4]

**Comment by BruceDone** [comment]
This framework is a  high-level browser automation , not a scrape or crawler framework.

**Comment by xcarpentier** [comment]
I know but :).

But I think it's also important to mention, maybe another section ? `high-level browser 
automation`

Because the main goal of scraping/crawling is to transform HTML to DATA, no. Such a thing you can do with browser automation tools like `nightmarejs` or `PhamtomJS` or `Selenium`...

**Comment by BruceDone** [comment]
that's right , other section like books ,demo or other things can really help to transform HTML to data will be good enough to be put in this .
